src/bucket/Bucket.h

```diff
@@ -132,6 +131,13 @@ class Bucket : public std::enable_shared_from_this<Bucket>,
     // entry in the database (respectively; if the entry is dead (a
     // tombstone), deletes the corresponding entry in the database.
     void apply(Application& app) const;
+
+    BucketIndex const&
```

Comment:
This is inside BUILD_TESTS. The reason I called it getIndexForTesting is because we already have a getIndex that's private. I just want to expose the index publicly for testing purposes, so I gave it a different name so the public and private versions wouldn't clash.

Reply:
(Marking resolved as Marta is on vacation and I think this has been addressed)